.github/workflows/build_package.yml

```diff
@@ -32,91 +32,56 @@ jobs:
       matrix:
         include:
           # Ubuntu packages.
-          - os: ubuntu-18.04
```

Comment:
Why are we dropping 18 completely or because of manylinux this isn't needed?

Reply:
may I suggest we use ubuntu-latest ? It is the latest LTS but then we anyway use manylinux2014 so we should be covered.

Reply:
Anush had just been prompting to move things forward. It doesn't matter much: nothing real actually runs on the host.